src/main/java/br/com/codex/mentoria/controller/AtuacaoController.java

```diff
@@ -0,0 +1,24 @@
+package br.com.codex.mentoria.controller;
+
+import br.com.codex.mentoria.model.dto.AtuacaoIn;
+import br.com.codex.mentoria.service.AtuacaoService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import static org.springframework.http.HttpStatus.CREATED;
+
+@RestController
+public class AtuacaoController {
+
+    @Autowired
```

Comment:
Se no projeto estiver sendo utilizado o lombok pode tirar o @Autowired e anotar assim **@RequiredArgsConstructor** @RestController
public class AtuacaoController {... e quando você for chamar a service você chama assim: private final AtuacaoService atuacaoService; pq ? Pq o Autowired é do spring então quando subimos nossa aplicação automáticamente ele injeta as dependencias mesmo se não formos usar, já o @RequiredArgsConstructor do lombok só injeta a dependencia se for necessário, no momento que alguém for usar aquela classe. Melhorando o desempenho da nossa aplicação sem sobrecarregar com algo que não está sendo usado.